### PR TITLE
[PORT] Adds Safety Moth posters from Beestation!

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
<details>
<summary>Ported from HERE:
</summary>
https://github.com/BeeStation/BeeStation-Hornet/pull/1747

(up-to-date sprites from https://github.com/BeeStation/BeeStation-Hornet/pull/1982)
</details>
<details><summary>Permission from all creators, including Aspev (Safetymoth Artists):</summary>
(Aspev's discord name censored for privacy)

aspeb

image

image
</details>
<hr>
Ports the Safety Moth posters from Beestation! Now crew will be able to read these informative safety posters, reminding them to always wear a hardhat, not cook meth without CMO approval, or not to stare into the SM (as pretty as it is).

**Examples**
<details><summary>
Hardhat</summary>

image
</details>
<details><summary>
Epipen</summary>

image
</details>
<details><summary>
Delam</summary>

image
</details>
Also includes a couple contraband posters of the same sort
<hr>

<details>
<summary>
**All posters in-game:**
</summary>

image
</details>

## Why It's Good For The Game
As said in the original Beestation PR:
1.  Giving the crew beneficial posters that promote safe workplace procedures.
2.  Safety Moth.
3.  More posters in general.

## Changelog

:cl:Orion_the_Fox(Porting), AyyRobotics(Original PR), AzlanonPC(Sprites), Aspev(SafetyMoth)
add: Added Safety Mothand a couple other similar posters.
imageadd: Updated contraband.dmi to add new poster icons.
/:cl:

### CODE REVIEW

**Comment by Farquaar** [comment]
These should all be contraband, in my opinion. I don't think NT would put non-humans centre-stage in their own propaganda.

**Comment by OrionTheFox** [comment]
> 
> 
> These should all be contraband, in my opinion. I don't think NT would put non-humans centre-stage in their own propaganda.

counter-point: what better way to make propaganda than to make it seem non-humans still agree?

**Comment by CRITAWAKETS** [comment]
Quite a bit of these look non-legible in pixel form, especially the contraband ones. Those ones just look like a pixel blob and unless you knew the source, i doubt you'd really discern what's even going on in those.

**Review by Mothblocks — APPROVED** [review]

File: code/game/objects/effects/contraband.dm
Comment (Mothblocks) [inline code/game/objects/effects/contraband.dm:642]: Actually I'm a bit weary of including the guy's username on these IC portraits

Also you should replace (TM) with just ™

**Comment by OrionTheFox** [comment]
> 
> 
> Quite a bit of these look non-legible in pixel form, especially the contraband ones. Those ones just look like a pixel blob and unless you knew the source, i doubt you'd really discern what's even going on in those.

aside from not being my sprites, they're made to be examined; theyre INFORMATIONAL, so hopefully if examined the pieces will be put together fairly fast

File: code/game/objects/effects/contraband.dm
Comment (OrionTheFox) [reply code/game/objects/effects/contraband.dm]: TM fixed, gonna hold off on the signature change until theres a tad more input on it

Comment (ike709) [reply code/game/objects/effects/contraband.dm]: AspEv asked for direct attribution when we added these to bee.

**Comment by SomeAngryMiner** [comment]
Most of these _are_ resized PNGs that were cleaned up, mind. I had no hand on the sprites whatsoever outside of advice and guidance on discord.

File: code/game/objects/effects/contraband.dm
Comment (OrionTheFox) [reply code/game/objects/effects/contraband.dm]: spebbv
asked 'em myself, they said its fine not having it directly in the descriptions; working out rn if they'd prefer A.E. or just to do it in code comments

Comment (OrionTheFox) [reply code/game/objects/effects/contraband.dm]: aaand they said just code comments are fine 😎

Comment (OrionTheFox) [reply code/game/objects/effects/contraband.dm]: done and committed; credits to AspEv are now in the code comments

**Comment by optimumtact** [comment]
Post some images of how they look in game please.

**Comment by Toastgoats** [comment]
Safety moth my beloved

**Comment by BlueMemesauce** [comment]
My opinion: 

- All of the posters could be changed have some sort of word at the top like most other posters, instead of black lines.
- The eyes of the delam poster blend in the background, the colors could be changed to fix that

**Comment by Farquaar** [comment]
The more I think about it, the safety poster should have a clown, not a moth. It's the closest think NT has to a mascot.
I'm too lazy to sprite it though.

**Comment by OrionTheFox** [comment]
> 
> 
> Post some images of how they look in game please.

image
will also be edited into the PR body as well

**Comment by OrionTheFox** [comment]
> 
> 
> My opinion:
> 
>     * All of the posters could be changed have some sort of word at the top like most other posters, instead of black lines.
> 
>     * The eyes of the delam poster blend in the background, the colors could be changed to fix that

while i'm not a spriter, i /can/ try to fix the delam poster issue, should just be a tiny tweak

**Comment by optimumtact** [comment]
they are a little busy but look okay to me

**Comment by OrionTheFox** [comment]
> 
> 
> they are a little busy but look okay to me

the hope for me, is that people examine them to read the legitimate safety tips they have; (plus, again, im not a spriter, im just porting these)

**Comment by Mothblocks** [comment]
my 2 cents: might be better with just a couple rather than all ten, but that's up to art maintainers not me

**Comment by OrionTheFox** [comment]
something has gone wrong

**Comment by OrionTheFox** [comment]
why did it do this noooo

**Comment by OrionTheFox** [comment]
okay i THINK i fixed whatever it just did

**Comment by OrionTheFox** [comment]
no i didnt :(
i'm gonna re-open it as a new PR with a few hand-picked posters rather than the whole batch

**Comment by LemonInTheDark** [comment]
The mass request + labeling was caused by you failing to merge properly, and commiting a revert of x amount of history back into the repo

**Comment by OrionTheFox** [comment]
> 
> 
> The mass request + labeling was caused by you failing to merge properly, and commiting a revert of x amount of history back into the repo
i know i know, i could have sworn i hit the right things, but alas x-x amateur coder moment
i've got a better version about ready to open that trims it to the most helpful info-posters though, so i'm just gonna open that one

**Comment by LemonInTheDark** [comment]
o7

**Comment by OrionTheFox** [comment]
https://github.com/tgstation/tgstation/pull/62936 for the new PR

**Pull request closed without merging**
